riscv/insns/c_fld.h

```diff
@@ -1,3 +1,4 @@
 require_extension(EXT_ZCD);
+require_extension('D');
```

Comment:
The load/store changes should be a separate commit.

Reply:
We can not simply enable Zcd when we set misa.D if we disable it when we clear misa.D in current implementation. There are two different cases: D_Zca_Zcd and D_Zca. So I do nothing for Zcd when misa.D changes and add these checks.